src/config/constants.js

```diff
@@ -17,6 +17,7 @@ export const ERROR_MESSAGE = Object.freeze({
   KEYWORD_EMPTY_INPUT_VALUE: "키워드를 입력해주세요.",
   CREATE_KEYWORD_ERROR: "새로운 키워드 생성에 실패하였습니다.",
   KEYWORD_DUPLICATED_INPUT_VALUE: "이미 키워드가 필터로 등록되어 있어요.",
+  APPLIED_POST_FILTER: "이미 필터가 적용되었어요.",
```

Comment:
```suggestion
  FILTER_ALREADY_APPLIED: "이미 필터가 적용되었어요.",
```
상수명만으로는 **이미** 적용되었단 의미가 포함되어 있지 않은 것 같아, `already` 단어가 들어가면 더 직관적일 것 같아요.

Reply:
이미가 더 명확해졌습니다, 감사해요!